packages/mds-db/policies.ts

```diff
@@ -267,6 +283,19 @@ export async function readRule(rule_id: UUID): Promise<Rule> {
   }
 }
 
+export async function findPoliciesByRule(rule_id: UUID): Promise<Policy[]> {
```

Comment:
We should make this only return a single policy instead of an array, especially if you're throwing an error upon the length != 1. That being said, I still think we could just use `readPolicies` and just add an extra param for `rule_id` and the conditional statement

Reply:
The error is only thrown if zero policies are returned. It doesn't say in the spec that rules must have unique UUIDs and there's nothing preventing multiple policies from sharing a rule at the moment.

Reply:
The error is thrown when `res.rowCount !== 1` not `res.rowCount === 0`. The `count` endpoint is really only valid under the assumption that `rule_ids` are unique, even though it's not explicit in the spec (or enforced in the implementation)